swift_browser_ui_frontend/wasm/README.md

```diff
@@ -0,0 +1,36 @@
+# Upload encryption based on crypt4gh using libcrypt4gh
+
+## Implementation details
+In order to keep the C side of things as simple as possible, we assume the
+correct directory structure to always be in place when the upload loop is run.
+The filenames are also kept constant – JS side is relied upon to copy the
+correct private etc. keys to the filesystem from browser storage.
+
+The directory structure is as follows:
+
+* keys
+    * pk.key
+    * recv_keys
+        * pub_key_0
+        * pub_key_1
+        * pub_key_2
+        * ...
+        * pub_key_n
+* data
+    * data_0
+    * data_1
+    * data_f
+        * data_2
+        * data_3
+    * data_4
+    * ...
+    * data_n
+
+The `recv_keys` file is simply a folder that contains multiple files, all of
```

Comment:
i think double ` would be more appropriate for this kind of formatting

Reply:
I'm not completely sure what you mean by this, there is no apparent difference between `single` and ``double`` backtics.